package.json

```diff
@@ -24,10 +24,10 @@
     "dayjs": "^1.11.6",
     "eslint-config-next": "^13.0.4",
     "is-mobile": "^3.1.1",
-    "next": "^13.0.4",
+    "next": "^14.2.11",
     "nextjs-progressbar": "^0.0.16",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
Please check, if we don't use these packages, we can delete them:

```
* eslint-config-next
* nextjs-progressbar
* react-toastify
* swr
```

Reply:
Done in https://github.com/rango-exchange/explorer/pull/32/commits/8f06dd6a784b6e366e920e679d16a1cf88404df0